lib/session/call-session.js

```diff
@@ -2838,6 +2839,24 @@ Duration=${duration} `
     if (Object.keys(opts).length > 0) {
       this.ep.set(opts);
     }
+
+    const origDestroy = this.ep.destroy.bind(this.ep);
+    this.ep.destroy = async() => {
+      try {
+        if (this.currentTask.name === TaskName.Transcribe && JAMBONES_TRANSCRIBE_EP_DESTROY_DELAY_MS) {
+          // transcribe task is being used, wait for some time before destroy
+          // if final transcription is received but endpoint is already closed,
+          // freeswitch module will not be able to send the transcription
+
+          this.logger.info('callSession:_configMsEndpoint -' +
+            ' transcribe task, wait for some time before destroy');
+          await new Promise((resolve) => setTimeout(resolve, JAMBONES_TRANSCRIBE_EP_DESTROY_DELAY_MS));
```

Comment:
I prefer to just add a `sleepFor` function at the top of the file and use that, it's more understandable
```
const sleepFor = (ms) => new Promise((resolve) => setTimeout(() => resolve(), ms));
```

Now I see that in dequeue.js we already did something similar.  Please move the function to utils so we can just require it anywhere needed

Reply:
fixed.